client/deployment/pom.xml

```diff
@@ -3,8 +3,9 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>io.quarkiverse.openapi.generator</groupId>
-    <artifactId>quarkus-openapi-generator-parent</artifactId>
+    <artifactId>quarkus-openapi-generator-client-parent</artifactId>
```

Comment:
I think we should keep the artifact name untouched for backward compatibilitty reasons and just add the server suffix to the new artifact containing the server stubs